inst/extdata/aglu/FAO_USA_an_Prod_t_PRODSTAT.csv

```diff
@@ -1 +1,60 @@
United States of America,231,"Other bird eggs,in shell",1091,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Pig meat,1035,Production (tonnes),51,5174000,5365000,5637000,5676000,5053000,5143260,5706620,5925700,5876260,6090000,6705000,6176600,5783730,6261810,5217650,5631350,6008740,6074960,7008000,7536440,7200740,6454160,6895500,6718600,6715400,6379000,6520000,7114000,7172000,6964000,7257500,7817000,7751000,8027000,8097000,7764000,7835000,8623000,8758000,8597000,8691000,8929000,9056000,9302730,9382560,9550390,9951360,10599000,10441700,10185600,10330700
United States of America,231,Rabbit meat,1141,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Sheep meat,977,Production (tonnes),51,377386,366954,349264,324316,295000,294834,293019,273061,249475,249928,251742,246299,233145,210919,185972,168282,159210,140159,132902,144240,153310,165560,170550,172360,162000,153300,143000,152000,157400,164650,164650,157850,152900,139700,130200,121560,118000,113900,112500,106100,101200,101000,92000,88450,84820,83910,82550,81693,80377,76340,76340
United States of America,231,"Sheep milk, whole, fresh",982,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Sheepskins,995,Production (tonnes),51,52611,51504,48441,44685,39900,39012,39105,36360,32768,32405,32895,31575,29397,27192,24141,20733,19664,16629,15567,17226,18591,19932,20376,20700,18901,17286,15936,16176,16677,17250,17439,16755,15888,15042,13884,12747,11907,11583,11298,10581,9870,10052,9127,8719,8288,8303,7584,7942,7834,7658,6773
United States of America,231,"Silk-worm cocoons, reelable",1185,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Skin Furs,1195,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Skins With Wool Sheep,999,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,"Snails, Not Sea",1176,Production (tonnes),51,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
United States of America,231,Turkey meat,1080,Production (tonnes),51,678207,586945,612800,659220,687189,759335,847991,730733,728465,784044,803922,865982,865500,857300,796000,914000,882700,908500,998500,1075000,1150000,1121300,1174800,1179800,1277800,1431000,1679000,1759500,1876000,2047500,2088000,2167000,2176000,2239000,2299000,2450000,2455000,2365000,2372000,2419000,2490000,2557000,2529000,2473890,2496570,2579130,2699260,2833680,2568810,2560210,2626580
United States of America,231,"Wool, greasy",987,Production (tonnes),51,144156,134838,127642,115666,109349,107058,102995,96273,88368,85017,81899,79377,71367,63989,54220,52590,49804,47150,47980,48290,50320,48140,46670,43395,39940,38270,38305,40588,40469,39931,39799,37622,35169,31106,28809,25700,24440,22334,21130,20662,19122,18633,17326,17046,16865,16284,15750,14952,13770,13776,13286
```

Comment:
This is strange. I did put the header,  and I can see them in all the input files.